src/pybind/mgr/dashboard/README.rst

```diff
@@ -69,6 +69,41 @@ The password will be stored as a hash using ``bcrypt``.
 
 The Dashboard's WebUI should then be reachable on TCP port 8080.
 
+Enabling the Rados Gateway management frontend
```

Comment:
I suggest to rename "Rados Gateway" to "Object Gateway", as that's how the [official documentation](http://docs.ceph.com/docs/master/radosgw/) refers to it.